### PR TITLE
update links

### PR DESCRIPTION
**Pull request by winsvega**
connects to https://github.com/ethereum/webthree-umbrella/issues/356

### CODE REVIEW

File: views/includes/footer.jade [hunk 1]
```diff
@@ -15,13 +15,11 @@ footer.scrollme
...
                 li Learn the  
                     a(href='https://solidity.readthedocs.org/') Solidity Language
-                li Learn more on the 
-                    a(href='https://github.com/ethereum/wiki/wiki') Main
-                    | , 
-                    a(href='https://github.com/ethereum/go-ethereum/wiki/') Go 
-                    | and 
-                    a(href='https://github.com/ethereum/webthree-umbrella/wiki/') C++ 
-                    | Wikis
+				li Learn more on the Ethereum
```
Comment (chriseth) [inline views/includes/footer.jade]: Indentation changed.

File: views/content/cli.md [hunk 2]
```diff
@@ -186,15 +186,15 @@ The console has auto completion and history support that persists between sessio
 
 #### Learn More on Running a node
 
-* [Backup and restore](https://ethereum-homestead.readthedocs.org/en/latest/using-ethereum-the-basics/backup-import-accounts.html)
-* [Connecting to the network](https://ethereum-homestead.readthedocs.org/en/latest/using-ethereum-the-basics/connecting-to-the-network.html)
+* [Backup and restore](https://ethereum.gitbooks.io/frontier-guide/content/backup_restore.html)
```
Comment (bobsummerwill) [inline views/content/cli.md]: These are regressions.   They were pointing to the right content, but through the wrong alias.   The should go to ethdocs.org.

File: views/content/cli.md [hunk 3]
```diff
@@ -186,15 +186,15 @@ The console has auto completion and history support that persists between sessio
...
+* [Connecting to the network](https://ethereum.gitbooks.io/frontier-guide/content/connecting.html)
```
Comment (bobsummerwill) [inline views/content/cli.md]: These are regressions.   They were pointing to the right content, but through the wrong alias.   The should go to ethdocs.org.

File: views/content/cli.md [hunk 4]
```diff
@@ -186,15 +186,15 @@ The console has auto completion and history support that persists between sessio
...
 
 
 ### Usage examples
 
 #### Creating accounts
 
-In order to do anything on an Ethereum network you need ether, and to get it, you will need to generate an account. There are [various ways to go around this](https://ethereum-homestead.readthedocs.org/en/latest/using-ethereum-the-basics/account-management.html), but the simplest one is through the console.
```
Comment (bobsummerwill) [inline views/content/cli.md]: These are regressions.   They were pointing to the right content, but through the wrong alias.   The should go to ethdocs.org.

File: views/content/cli.md [hunk 5]
```diff
@@ -252,7 +252,7 @@ Once you executed the line above, all you need to check all of your balances is
 
 #### Learn more
 
-* [Account Types, Gas, and Transactions](https://ethereum-homestead.readthedocs.org/en/latest/contracts-and-transactions/account-types-gas-and-transactions.html)
-* [Developing on Ethereum](https://ethereum-homestead.readthedocs.org/en/latest/developing-on-ethereum/dapp-development.html)
-* [Developer tools](https://ethereum-homestead.readthedocs.org/en/latest/developing-on-ethereum/developer-tools.html)
-* [Solidity Documentation](https://solidity.readthedocs.org/)
+* [Account Types, Gas, and Transactions](https://ethereum.gitbooks.io/frontier-guide/content/contracts_and_transactions_intro.html)
```
Comment (bobsummerwill) [inline views/content/cli.md]: These are regressions.   They were pointing to the right content, but through the wrong alias.   The should go to ethdocs.org.

File: views/content/cli.md [hunk 6]
```diff
@@ -252,7 +252,7 @@ Once you executed the line above, all you need to check all of your balances is
...
+* [Developing on Ethereum](https://ethereum.gitbooks.io/frontier-guide/content/writing_contract.html)
```
Comment (bobsummerwill) [inline views/content/cli.md]: These are regressions.   They were pointing to the right content, but through the wrong alias.   The should go to ethdocs.org.

**Comment by bobsummerwill** [comment]
Thanks, @winsvega.

So it looks like the only Frontier-guide stuff we have remaining is for Go, right?

**Pull request merged**